src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -92,6 +90,11 @@ public static String getBrowser() {
     return getProp("browser");
   }
 
+  public static final String WIKI_NAME_PROPERTY = "wikiName";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)